api/v1alpha2/jenkins_webhook.go

```diff
@@ -184,44 +186,42 @@ func Validate(r Jenkins) error {
 	return nil
 }
 
-func NewPluginsDataManager(compressedFilePath string, pluginDataFile string, isCached bool, timeout time.Duration) *PluginDataManager {
-	return &PluginDataManager{
-		CompressedFilePath: compressedFilePath,
-		PluginDataFile:     pluginDataFile,
-		IsCached:           isCached,
-		Timeout:            timeout,
+// NewMonitor creates a new worker and instantiates all the data structures required.
+func NewSecurityValidator() *SecurityValidator {
+	return &SecurityValidator{
+		isCached:       false,
+		Attempts:       0,
+		checkingPeriod: shortenedCheckingPeriod,
 	}
 }
 
-func (in *PluginDataManager) ManagePluginData(sig chan bool) {
-	var isInit bool
-	var retryInterval time.Duration
+func (in *SecurityValidator) MonitorSecurityWarnings(securityWarningsFetched chan bool) {
```

Comment:
```suggestion
func (s *SecurityValidator) MonitorSecurityWarnings(securityWarningsFetched chan bool) {
```
I think it would fit go convetions better

Reply:
It won't work in webhook file.